src/components/forecasts-list.js

```diff
@@ -0,0 +1,73 @@
+import { useSelector } from "react-redux";
+import { Sparklines, SparklinesLine, SparklinesReferenceLine } from "react-sparklines";
+import _ from 'lodash';
+
+const CityForecastsList = () => {
+  const forecasts = useSelector((state) => state.forecasts);
+
+  // console.log(forecasts);
+
+  const renderForecasts = () => {
+    if (!_.isEmpty(forecasts.queryCityEntries)) {
+      return forecasts.queryCityIds.map((id) => {
+        return (
+          <tr key={id}>
+            <td>{forecasts.queryCityEntries[id].city}</td>
+            <td>
+              <Sparklines data={forecasts.queryCityEntries[id].temperature} svgwidth={150} svgheight={100}>
+                <SparklinesLine color='purple'/>
+                <SparklinesReferenceLine type='mean'/>
+              </Sparklines>
+              <div>{Math.round(forecasts.queryCityEntries[id].temperature.reduce((a, b) => a + b) / forecasts.queryCityEntries[id].temperature.length)} &deg; F</div>
+            </td>
+            <td>
+              <Sparklines data={forecasts.queryCityEntries[id].pressure} svgwidth={150} svgheight={100}>
+                <SparklinesLine color='orange'/>
+                <SparklinesReferenceLine type='mean'/>
+              </Sparklines>
+              <div>{Math.round(forecasts.queryCityEntries[id].pressure.reduce((a, b) => a + b) / forecasts.queryCityEntries[id].pressure.length)} hPa</div>
```

Comment:
Seeing a lot of repetition, could possibly extra this out into a helper function